packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -79,6 +86,13 @@ import logger from './logger'
  * @property {string} _type - Doctype of the client (i.e. io.cozy.oauth.clients)
  */
 
+/**
+ * Options that can be passed to FileCollection's constructor
+ *
+ * @typedef {object} FileCollectionOptions
+ * @property {string} [driveId] - ID of the shared drive targeted by the collection
```

Comment:
Do we have some doc about the shared drive somewhere? Typically from the stack side?

Reply:
We do have documentation in `cozy-stack` about shared drives.

Reply:
I was wondering why I didn't find any code :) 
 Ok, so the stack's doc link could be added to the jsdoc eventually